hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/util/Util.java

```diff
@@ -442,8 +443,21 @@ public static String addOrIncrementIndexInName(String name) {
         return name + '-' + index;
     }
 
-    public static String sanitizeLoggerNamePart(String name) {
-        return name.replace('.', '_');
+    public static ILogger prefixedLogger(ILogger logger, String prefix) {
+        return new PrefixedLogger(logger, prefix);
+    }
+
+    public static String prefix(String jobName, long jobId, String vertexName) {
+        return prefix(jobName, jobId, vertexName, null);
+    }
+
+    public static String prefix(String jobName, long jobId, String vertexName, int processorIndex) {
+        return prefix(jobName, jobId, vertexName, "procIndex:" + processorIndex);
```

Comment:
IMO we can omit the `procIndex`:
```suggestion
        return prefix(jobName, jobId, vertexName, processorIndex + "");
```